pkg/diagnostics/collectors.go

```diff
@@ -317,9 +317,9 @@ func (c *collectorFactory) snowCrdCollectors() []*Collect {
 
 func (c *collectorFactory) tinkerbellCrdCollectors() []*Collect {
 	captCrds := []string{
-		"baseboardmanagements.bmc.tinkerbell.org",
-		"bmcjobs.bmc.tinkerbell.org",
-		"bmctasks.bmc.tinkerbell.org",
+		"machines.bmc.tinkerbell.org",
```

Comment:
Did we rename this CRD to `machines` upstream?
CAPI already has a CRD named `machines` so this is gonna be ambiguous

Reply:
we did. https://github.com/tinkerbell/rufio/blob/main/config/crd/bases/bmc.tinkerbell.org_machines.yaml
It's namespaced though.

Reply:
I agree that a CRD named `machines` is going to be confusing and ambiguous

Reply:
Custom resources are namespaced so there aren't any collisions from a technical point of view; I can appreciate humans may do a double take. There are numerous projects that define the same kind across different groups (namespaces). For example, [argo-cd](https://doc.crds.dev/github.com/argoproj/argo-cd) and [litmus](https://doc.crds.dev/github.com/litmuschaos/litmus) define `Workflow` and `CronWorkflow` types. [Crossplane](https://doc.crds.dev/github.com/crossplane/crossplane) defines 2 `Configuration` resources in the same project but under different namespaces. There are many other examples.

Its infeasible, and defeats the point of namespacing, to expect projects to need to create unique names - not to mention this is upstream so can't be changed here.